CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- `opentelemetry-instrumentation-aiohttp-server`, `opentelemetry-instrumentation-httpx`, `opentelemetry-resource-detector-azure` Ensure consistently use of suppress_instrumentation utils; adapt `opentelemetry.instrumentation.utils` util functions `suppress_instrumentation` and `is_instrumentation_enabled` to also check the old plain string key "suppress_instrumentation" for backward compatibility
```

Comment:
azure detector has its own changelog, you should update that too and just list instrumentation bumps there

Reply:
Missed this. Thanks